apps/web/components/booking/CancelBooking.tsx

```diff
@@ -23,6 +23,7 @@ type Props = {
   theme: string | null;
   allRemainingBookings: boolean;
   seatReferenceUid?: string;
+  currentUserEmail?: string | null;
```

Comment:
`CancelBooking ` is being used in bookings/{bookingUid} page where we get currentUserEmail either from session user or searchParams where searchParams?.get("email") resolves to null in case email is not present as params. You can check the code block you reivewed below, that is where Cancel Booking component is used.

Reply:
```suggestion
  currentUserEmail: string;
```

Reply:
Since we are defaulting to anonymous we can change it to string